src/pocketdb/repositories/web/ExplorerRepository.cpp

```diff
@@ -121,6 +121,98 @@ namespace PocketDb {
         return result;
     }
 
+    UniValue ExplorerRepository::GetContentStatisticByHours(int topHeight, int depth)
+    {
+        UniValue result(UniValue::VOBJ);
+
+        TryTransactionStep(__func__, [&]()
+        {
+            auto stmt = SetupSqlStatement(R"sql(
+                select (u.Height / 60)
+                  ,(
+                    select
+                      count()
+                    from Transactions u1 indexed by Transactions_Type_Last_Height_Id
+                    where u1.Type in (100)
+                    and u1.Height <= u.Height
+                    and u1.Last = 1
+                  )cnt
+                from Transactions u indexed by Transactions_Type_HeightByHour
+                where u.Type in (3)
+                  and (u.Height / 60) <= (? / 60)
```

Comment:
I can't get how it mixes here  depth that seems to represent time and height.

Reply:
Blocks are generated once a minute, +- several minutes. Therefore, we take the height as the number of minutes - the error in this case (the explorer graph) is not very important